src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements.mdx

```diff
@@ -51,15 +51,25 @@ Make sure your Android app meets these requirements:
       <td>
 HttpURLConnection, OkHttp2 (except versions 2.0 and 2.4), OkHttp (versions 2.8, 3.5+, 4.0+), OkIO (version 1.11), AndroidHttpClient, Volley (version 1.0.0), and Apache HTTP Client networking APIs
 
-<Callout variant="important">
+  <Callout variant="important">
   AndroidHttpClient was removed from the SDK in version 23, and as such only supported in Android agent version 6.5.0 and lower.
   </Callout>
 
+**Android agent version 7.0.0 or higher** requires:
+
+* Android Studio Flamingo
+* Gradle 7.1
+* Android Gradle plugin 7.0.0
+* DexGuard 9.0 and higher
+  <Callout variant="important">
+  DexGuard 9 doesn't yet support Gradle 8 or AGP 8
```

Comment:
You know, let's just pull the call-out. I haven't rechecked the status of DG9 and Gradle 8, and we'd have to remove this anyway when it is supported.

Reply:
Resolved